ExampleApp/AppDelegate.swift

```diff
@@ -67,6 +69,64 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 
     // MARK: Private
 
+    private func registerNotificationCenterObservers() {
+        NotificationCenter.default.addObserver(
+            self,
+            selector: #selector(didLink),
+            name: TradeItSDK.didLinkNotificationName,
+            object: nil
+        )
+
+        NotificationCenter.default.addObserver(
+            self,
+            selector: #selector(didUnlink),
+            name: TradeItSDK.didUnlinkNotificationName,
+            object: nil
+        )
+
+        NotificationCenter.default.addObserver(
+            self,
+            selector: #selector(onViewDidAppearNotification),
+            name: TradeItNotification.Name.viewDidAppear,
+            object: nil
+        )
+
+        NotificationCenter.default.addObserver(
+            self,
+            selector: #selector(onErrorShownNotification),
+            name: TradeItNotification.Name.errorShown,
+            object: nil
+        )
+    }
+
+    func onErrorShownNotification(notification: Notification) {
+        let view = notification.userInfo?[TradeItNotification.UserInfoKey.view] ?? "NO KEY FOR VIEW"
+        let errorTitle = notification.userInfo?[TradeItNotification.UserInfoKey.errorTitle] ?? "NO KEY FOR ERROR CODE"
+        let errorMessage = notification.userInfo?[TradeItNotification.UserInfoKey.errorMessage] ?? "NO KEY FOR ERROR MESSAGE"
+        print("=====> ERROR SHOWN: VIEW: \(view), TITLE: \(errorTitle), MESSAGE: \(errorMessage)")
```

Comment:
Do we want to just pass the error object back here?

Reply:
That makes sense.